cli/internal/cmd/check_test.go

```diff
@@ -62,9 +65,14 @@ func TestCheckDeploymentStatus(t *testing.T) {
 	assert := assert.New(t)
 	require := require.New(t)
 	testClient := fake.NewSimpleClientset()
+	ctx := context.Background()
+	cmd := &cobra.Command{}
+	cmd.SetContext(ctx)
```

Comment:
Is this still required for testing? I thought I've fixed it upstream a while ago...